packages/components/src/Icon/Icon.tsx

```diff
@@ -6,11 +6,11 @@ import { createCssClassNames } from '@ids-internal/shared/css.class.names';
 
 import { IconProps } from './Icon.types';
 
-const Icon = ({ cssClass = '', name, customPath, size = undefined }: IconProps) => {
+const Icon = ({ cssClass = '', name, customPath, size = 'small' }: IconProps) => {
```

Comment:
adjust to other components that doesn't allow for undefined values, "default" (without size modifier) is same as "small" in css